tasks/redhat.yml

```diff
@@ -0,0 +1,20 @@
+- name: Add firewalld rules
```

Comment:
Can you please make this conditional. Some systems may already have existing firewall configurations for this and not wish to modify their rules. This could be conditionally included from the `main.yaml` file after renaming this file to `firewall.yml`.

Reply:
zookeeper_firewalld is new var for this